src/commands/cmd_info.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Redis Ltd. 2018 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+ /* This file contains the implementation of the GRAPH.INFO command. */
+
+#include "commands.h"
+#include "redismodule.h"
+
+// Dispatch the subcommand.
```

Comment:
Please add a comment providing an example user invocation to this function, e.g.
`GRAPH.INFO g`

Reply:
Then please include a few examples of commands that will get us here, this becomes really convenient when debugging or just to get into context.